index.html

```diff
@@ -0,0 +1,405 @@
+                alt="Resume image" class="sugImg">
+                <figcaption class="sugImgtext"><label class="imgctext">  </label></figcaption>
+                <label class="article sugTitle"><a href="#" class="sugLink">Résumé-Writing Tips to Help You Get Past the A.I. Gatekeepers</a></label>
+                <label class="imgctext articleinfo datepublished">March 19</label>
+            </figure>
+        </section>
+    </section>
+    <aside class="popular">
+        <p class="articleinfo"><strong>Most Popular</strong></p>
+        <ul class="pplr">
+            <li><a href="#" class="sugLink"><p class="article content">A Man Demanded His Final Paycheck. The Auto Shop Delivered 
+                91,500 Greasy Pennies.</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Presidential Challenger Dies of Covid-19 Just Hours After Voting Ends</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Woman Is Fatally Stabbed by Her Son During Video Call, Authorities Say</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Sharon Stone Is Telling Her Side of the Story</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Evils Lurking Beneath a City’s Centuries-Old Canals</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">An E.R. Doctor Wanted a Quiet Place for Less Than $2,200. Which Home Would You Choose?</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Divorce Rates Are Now Dropping. Here Are Some Reasons Why.</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Julie Pomagalski, Former Olympic Snowboarder, Is Killed in an Avalanche</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">Welcome to Paradise. You May Want to Catch the Next Flight Home.</p></a></li>
+            <li><a href="#" class="sugLink"><p class="article content">‘Six Minutes to Midnight’ Review: A Finishing School for the Nazi Elite</p></a></li>
+        </ul>
+    </aside>
+</section>
+<footer class="additionalinfo imgctext">
+    <p>© 2021 The New York Times Company</p>
+    <nav class="foot">
+        <a href="#" class="addlink">NYTCo</a>
+        <a href="#" class="addlink">Contact Us</a>
+        <a href="#" class="addlink">Accessiblity</a>
+        <a href="#" class="addlink">Work with us</a>
+        <a href="#" class="addlink">Advertise</a>
+        <a href="#" class="addlink">T Brand Studio</a>
+        <a href="#" class="addlink">Your Ad Choices</a>
+        <a href="#" class="addlink">Privacy Policy</a>
+        <a href="#" class="addlink">Terms of Service</a>
+        <a href="#" class="addlink">Terms of Sale</a>
+        <a href="#" class="addlink">Site Map</a>
+        <a href="#" class="addlink">Help</a>
+        <a href="#" class="addlink">Subscriptions</a>
+    </nav>
+</footer>
```

Comment:
## Implement missing footer section
- [ ] This section of the footer is missing. Do you mind implementing it?  The sticky part of the footer is optional but it would be nice to have.
![footer](https://user-images.githubusercontent.com/22345961/112553752-2187d580-8dd6-11eb-94dc-753442e649b0.png)